journal.py

```diff
@@ -145,18 +170,37 @@ def editor(request):
     else:
         raise HTTPUnauthorized
 
+
 @view_config(route_name='delete')
 def delete(request):
     param = request.matchdict.get('id', -1)
     request.db.cursor().execute(DELETE_ENTRY, [param])
     return HTTPFound(request.route_url('home'))
 
+
 @view_config(route_name='details', renderer="templates/details.jinja2")
 def details(request):
     entry = get_entry(request)
     entry[0]['text'] = markdown.markdown(entry[0]['text'], extensions=('codehilite', 'fenced_code'))
     return {'entries': entry}
 
+
+@view_config(route_name='tweet', renderer='json')
```

Comment:
This doesn't actually return anything.  I would expect the json renderer to throw an error.  What actually happens?